setup.cfg

```diff
@@ -76,12 +76,13 @@ encryption =
 
 [options.entry_points]
 console_scripts =
+  pyi = PyInstaller.__main__:run
 	pyinstaller = PyInstaller.__main__:run
-    pyi-archive_viewer = PyInstaller.utils.cliutils.archive_viewer:run
-    pyi-bindepend = PyInstaller.utils.cliutils.bindepend:run
-    pyi-grab_version = PyInstaller.utils.cliutils.grab_version:run
-    pyi-makespec = PyInstaller.utils.cliutils.makespec:run
-    pyi-set_version = PyInstaller.utils.cliutils.set_version:run
+  pyi-archive_viewer = PyInstaller.utils.cliutils.archive_viewer:run
+  pyi-bindepend = PyInstaller.utils.cliutils.bindepend:run
+  pyi-grab_version = PyInstaller.utils.cliutils.grab_version:run
+  pyi-makespec = PyInstaller.utils.cliutils.makespec:run
+  pyi-set_version = PyInstaller.utils.cliutils.set_version:run
```

Comment:
I'd rather we just got rid of these. I think it defeats the purpose of using subcommands to give us a single point of entry. I'm not a fan of the `pyi` shorthand either.

Reply:
I do plan to remove these. Only `pyi` and `pyinstaller` will remain, and the two will be identical. As for the shorthand, I personally like it and don't see the harm in having both.